cpp/ql/test/library-tests/lambdas/captures/elements.expected

```diff
@@ -56,11 +58,13 @@
 | captures.cpp:9:9:9:9 | operator() |
 | captures.cpp:9:9:11:5 | { ... } |
 | captures.cpp:10:7:10:7 | (captured this) |
+| captures.cpp:10:7:10:7 | (captured this) |
 | captures.cpp:10:7:10:7 | call to b |
 | captures.cpp:10:7:10:7 | definition of (captured this) |
 | captures.cpp:10:7:10:15 | ExprStmt |
 | captures.cpp:10:9:10:9 | definition of x |
 | captures.cpp:10:9:10:9 | x |
+| captures.cpp:10:9:10:9 | x |
```

Comment:
I don't understand why a second copy of something that stringifies as `x` appeared as a result of this change. This is on line 10, where there's no lambda capture, so why is there a new element here? Is something funny going on with the locations of these lambda captures?

Reply:
This particular test doesn't stringify, it outputs each `Element` and in this case there are two - a `Field` and a `LambdaCapture`.  Previously we wouldn't have output the `LambdaCapture` but only because it wasn't an `Element`.

I'm not a fan of tests that output all `Element`s, I could perhaps modify it to only output a fixed list of things (perhaps `Variable`, `VariableDeclarationEntry`, `ExprStmt` and `LambdaCapture`) along with what they are?

Reply:
I think my confusion is that the capture and definition of `x` ought to belong on line 9, which reads `[=] {`, rather than line 10, which reads `b(x + 1);`. But maybe that's just an off-by-one somewhere. It's certainly not related to this PR.

Now that we have `getCanonicalQLClass`, it should be easier to write those tests that stringify a set of class names. But that's not for this PR.